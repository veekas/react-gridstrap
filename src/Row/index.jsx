import React from 'react';
import PropTypes from 'prop-types';

const Row = ({ children, className, ...props }) => {
  return (
    <div className={className} {...props}>
      {children}
    </div>
  );
}

Row.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string,
};

Row.defaultProps = {
  children: null,
  className: '',
};
