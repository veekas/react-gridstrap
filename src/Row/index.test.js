import React from 'react';
import ReactDOM from 'react-dom';
import Row from './';

describe('Row', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Row />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});
