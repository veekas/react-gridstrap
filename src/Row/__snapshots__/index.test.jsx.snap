// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row matches snapshot when passed zero props 1`] = `
<div
  className="css-dtn8f5"
/>
`;
